lib/db.js

```diff
@@ -289,17 +289,24 @@ class LimitDBRedis extends EventEmitter {
   }
 
   takeElevated(params, callback) {
-    const valError = validateERLParams(params.elevated_limits);
-    if (valError) {
-      return callback(valError)
+    const valERLParamsError = validateERLParams(params.elevated_limits);
+    if (valERLParamsError) {
+      return callback(valERLParamsError)
     }
     const erlParams = utils.getERLKeysQuotaAmountAndExpiration(params.elevated_limits);
 
     this._doTake(params, callback, (key, bucketKeyConfig, count) => {
-      const valError = validateConfigIsForElevatedBucket(key, bucketKeyConfig)
-      if (valError) {
-        return callback(valError)
+      let isERLEnabledForBucket = true;
+      const valERLConfigError = validateConfigIsForElevatedBucket(key, bucketKeyConfig)
+      if (valERLConfigError) {
```

Comment:
is this semantically different than going down the `take()` function at this point?

Reply:
sorry, i know there was a slack thread on this but I found it hard to parse 😅

Reply:
I suppose the return values are different?

Reply:
yes. If we make it go the `take()`, we will only be using `takeElevated.lua` for buckets with elevated limits configuration and we'll always depend on both scripts.
This way we'll be making sure `takeElevated.lua` works for both scenarios and in the future we'll be able to join them in a single method if we wish to do so.